plugin/storage/factory_config.go

```diff
@@ -69,11 +78,22 @@ func FactoryConfigFromEnvAndCLI(args []string, log io.Writer) FactoryConfig {
 	if depStorageType == "" {
 		depStorageType = spanWriterTypes[0]
 	}
+	ratioEnvVariable := os.Getenv(DownSamplingRatio)
+	// Default ratio to drop spans is 0
+	downSamplingRatio, err := strconv.ParseFloat(ratioEnvVariable, 64)
+	if err != nil {
+		downSamplingRatio = 0
+	}
+	// Default salt is empty string
+	downSamplingHashSalt := os.Getenv(DownSamplingHashSalt)
```

Comment:
provide a default.

Reply:
default is empty string, this case will be covered by err checking from `strconv.ParseFloat`